aiida/tools/archive/implementations/sqlite/backend.py

```diff
@@ -202,7 +202,7 @@ def maintain(self, dry_run: bool = False, live: bool = True, **kwargs) -> None:
         raise NotImplementedError
 
     def get_info(self, statistics: bool = False, **kwargs) -> dict:
-        raise NotImplementedError
+        return {'objects': {'count': len(list(self.list_objects()))}}
```

Comment:
You still forgot to add the `super` here no? Oh bah, were you not finish with the modifications maybe?

Reply:
It shouldn't have one. Note this is the repository implementation, the base abstract class of which doesn't define a `get_info`. Only the implementation for the disk object store does.

The base class with a `get_info` implementation is the one for the *storage backend*. I think you may be confusing it with that

Reply:
Ahhh ok, I see. Then my previous comment was also wrong, haha. Ok, all good now then.

Reply:
You were confused 😉 you put `get_info` and `maintain` under `SqliteBackendQueryBuilder` not `SqliteZipBackend`
Its ok, I fixed it in #5375

Reply:
You were confused 😉 you put `get_info` and `maintain` under `SqliteBackendQueryBuilder` not `SqliteZipBackend`
Its ok, I fixed it in #5375